webapp/channels/src/utils/constants.tsx

```diff
@@ -1071,6 +1071,7 @@ export const CloudLinks = {
 
 export const HostedCustomerLinks = {
     BILLING_DOCS: 'https://mattermost.com/pl/how-self-hosted-billing-works',
+    SELF_HOSTED_BILLING: 'https://docs.mattermost.com/manage/self-hosted-billing.html',
```

Comment:
Add this as its a more general page we want to redirect to. `HostedCustomerLinks.BILLING_DOCS` is a separate, more detailed page.